web/component/ads/view.jsx

```diff
@@ -144,7 +145,7 @@ function Ads(props: Props) {
   );
 
   if (type === 'video') {
-    if (shouldShowAds) {
+    if (shouldShowAds && userIsInUS) {
```

Comment:
Perhaps the mistake is in `resolveAdVisibility`, and should be corrected there? `shouldShowAds` should cover all scnearios.

Reply:
I don't know. Your solution seems to work here. I'm not quite sure what's happening. Anthony mentioned that he sees the ads again. This PR seems unnecessary and I might delete it. I'm just trying to understand why it isn't working for him...

Reply:
My bad. I misread that line.

Reply:
Throwing out ideas:
`export const selectUserCountry = createSelector(selectUserLocale, (locale) => locale?.country);`
means `country` could be `undefined` until fetched, so `undefined !== 'US'` results in `true`?

Reply:
Ah, never mind, saw the update.

Reply:
Should be fixed now :D Gonna test all the potential scenarios...